### PR TITLE
Generalise README.md

### PR DESCRIPTION
**Pull request by guillaume-roche**
Now that this repository is public, we can make the README.md more generic to make it clearer for third-party developers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
-See : https://android.googlesource.com/platform/build/+/refs/tags/android-10.0.0_r25/core/Makefile#306
+`dev-keys` are used in `device.mk` to sign the system at build (properties are flagged with dev-keys).
+We use `dev-keys/testkey`, but the build system flags them as `dev-keys`, as they are not the default AOSP keys.
 
-release-keys/ is used with signature script to resign as release (properties are flagged with release-keys, apk are resigned).
+See: https://android.googlesource.com/platform/build/+/refs/tags/android-10.0.0_r25/core/Makefile#306
+
+`release-keys` are used by signature script to re-sign the build as a release one. Properties are flagged as `release-keys`, apk files are re-signed.
 
 ## Key generation
```
Comment (lilinor) [inline README.md:16]: Do we need to keep this as users are not building Genymotion images?

File: README.md [hunk 2]
```diff
@@ -1,16 +1,17 @@
-# Genymotion dev and release keys.
+# Genymotion development and release keys
 
 Copyright (C) Genymobile
 
 ## Description
 
-Repository to store dev and release keys, and corresponding .mk that are included from device.mk.
+This repository stores Genymotion Android development and release keys, and corresponding .mk that are included from device.mk.
```
Comment (lilinor) [inline README.md]: Are .mk useful for end users because it points to vendor/genymotion that we don't provide anywhere?

Comment (lilinor) [reply README.md]: Maybe add a sentence after this one to describe the use case for using those keys. Something like :
Third party applications signed with those keys can gain system privileges.

Please add other use cases if there are any.

**Review by lilinor — CHANGES_REQUESTED** [review]
I made comments :)

File: README.md (hunk 1 above)
Comment (guillaume-roche) [reply README.md:16]: Maybe I can add something like: "internal use"

File: README.md (hunk 2 above)
Comment (guillaume-roche) [reply README.md]: Will do!

**Review by lilinor — APPROVED** [review]
Ok for me

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ Copyright (C) Genymobile
 
 ## Description
 
-This repository stores Genymotion Android development and release keys, and corresponding .mk that we include internally in our `device.mk`. Third party  developpers can use the release keys to sign their own applications in order to grant them system privileges.
+This repository stores Genymotion Android development and release keys, and corresponding .mk that we include internally in our `device.mk`. Third party developpers can use the release keys to sign their own applications in order to grant them system privileges on Genymotion. Only our Android 10 version rely on these keys for now.
```
Comment (lilinor) [inline README.md:7]: Can we add a note on which keys to use for the other Android versions?

**Pull request merged**